src/scss/box.scss

```diff
@@ -118,7 +128,7 @@
         overflow-x: auto;
     }
 
-    .row-label, label {
+    .row-label, label:not(.label-hack) {
```

Comment:
Let's rename our "`.label-hack` in a `box`" 😏

Reply:
Addressed above